talos.nf

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl=2
```

Comment:
dsl2 should be the default as of Nextflow 22 (I believe), so you don't need this line. Unless you are wanting to support older versions of Nextflow...

Reply:
I don't want to explicitly support more than I have to, but this model relies on the user having nextflow installed locally, which could be old or new